AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.java

```diff
@@ -172,7 +172,7 @@
     public static final String EXTRA_TAGS = "TAGS";
     public static final String EXTRA_ID = "ID";
     public static final String EXTRA_DID = "DECK_ID";
-
+    public static final String TEXT_FROM_SEARCH_VIEW= "SEARCH";
```

Comment:
Also: rename to prefix with the word `EXTRA`:

`EXTRA_FIRST_FIELD` for example

```suggestion
    public static final String TEXT_FROM_SEARCH_VIEW = "SEARCH";
```